airbyte-webapp/README.md

```diff
@@ -8,6 +8,14 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `npm ci`
+
+Installs essential modules Ex. "orval"
+
+### `npm install`
+
+Installs requirements.
```

Comment:
I'd suggest being a bit more elaborate here and saying something like "Install dependencies to build and run the webapp."